src/ToDoList.js

```diff
@@ -176,11 +179,11 @@ const ToDoList = () => {
         </Typography>
         <Typography variant="body1" fontWeight="bold">
           Completed Tasks:{" "}
-          {newTaskValue.filter((task) => task.completed).length}
+          {newTaskValue.filter((task) => task.isCompleted).length}
```

Comment:
If `newTaskValue.filter((task) => task.isCompleted)` is repeated multiple times, we'll want to store it into a variable once, and just use it everywhere it's needed, for increased performance.

Reply:
Ok, done.